lms/urls.py

```diff
@@ -1031,3 +1031,9 @@
 urlpatterns += [
     path('api/instructor_task/', include('lms.djangoapps.instructor_task.rest_api.urls')),
 ]
+
+# MFE API urls
+if settings.FEATURES.get('ENABLE_MFE_API'):
```

Comment:
Is it the best pattern to keep this behind the feature flag? I would be more on the side of making this always available and leave the check inside of the view. This way it can also be altered via tenancy models or site_configs.

Reply:
You're right, I am going to make that change.